docs/extensibility/extending-blocks.md

```diff
@@ -163,6 +163,20 @@ var withDataAlign = wp.compose.createHigherOrderComponent( function( BlockListBl
 wp.hooks.addFilter( 'editor.BlockListBlock', 'my-plugin/with-data-align', withDataAlign );
 ```
 
+### `editor.PostFeaturedImage.imageSize`
+
+Used to modify the image size displayed in the Post Featured Image component. It defaults to `'post-thumbnail'`, and will fail back to the `full` image size when the specified image size doesn't exist on the media object. It's modeled after the `admin_post_thumbnail_size` filter in the Classic Editor.
```

Comment:
I think "exist on the media object" should be "exist *in* the media object".

Reply:
328a08bc66afcc3545033f228eb7c9299420fde3